plugins/discover.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: discover
+spec:
+  version: v0.1.5
+  homepage: https://github.com/mateimicu/kdiscover
+  shortDescription: "Discover Kubernetes clusters and export configs"
```

Comment:
I think this doesn't make it clear what the plugin exactly is meant for (cloud). Maybe:

> Find/export kubeconfigs for cloud clusters

What do you think?